.github/workflows/main.yml

```diff
@@ -202,7 +202,16 @@ jobs:
     needs: build
     strategy:
       matrix:
+        k8s_version: [""]
+        focus: [""]
         bucket: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        include:
+          - k8s_version: v1.21.10
+            focus: "Test traffic flowing from client to server with a Kubernetes Service for the Source: HTTP"
+            bucket: ".*"
+          - k8s_version: v1.22.7
+            focus: "Test traffic flowing from client to server with a Kubernetes Service for the Source: HTTP"
+            bucket: ".*"
```

Comment:
Couldn't this be abbreviated with the following?

```yaml
matrix:
  k8s_version: ["v1.21.10", "v1.22.7"]
  focus: ["Test traffic flowing from client to server with a Kubernetes Service for the Source: HTTP"]
  bucket: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

I don't think there's anything gained by expanding the matrix config here with `include`.

Reply:
Ah I see, thanks for the clarification.